gsy_framework/sim_results/scm/bills.py

```diff
@@ -61,7 +61,7 @@ def update(self, area_result_dict, core_stats, current_market_slot):
                 self.bills_redis_results[area["uuid"]] = self._empty_bills_dict()
 
             area_bills = {
-                k: v + core_stats[area["uuid"]]["bills"].get(k, 0.)
+                k: (v or 0.) + (core_stats[area["uuid"]]["bills"].get(k, 0.) or 0.)
```

Comment:
Thanks for fixing this, but isn't this related to the new `KPICalculationHelper` class that we've added recently? Maybe we can avoid a problem here by fixing that. I'm suspecting this because we had a similar problem two weeks ago.

Reply:
I do not think so actually, at least from a brief look at the scm/billls.py module. This method is called directly by the ResultsHandler class, and its input is the data that are generated by the gsy-e IIRC. I suspect that the main reason for this popping up now is a change in gsy-e (where we had similar discussions about null values IIRC). 
Having said the above, I believe that this change is still valid, since no matter whether we receive null values here, the code should not crash but rather handle it appropriately.

Reply:
Thanks for the clarification.